.github/workflows/Helix-CI.yml

```diff
@@ -12,14 +12,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
     - name: Build with Maven
-      run: mvn clean install -Dmaven.test.skip.exec=true -DretryFailedDeploymentCount=5
```

Comment:
May I ask any reason for removing this "re-run failed tests" flag?

Reply:
i accidently had removed the modification. re-introduced with latest change